osidb/tests/factories.py

```diff
@@ -513,33 +513,35 @@ class Meta:
         model = Snippet
 
     class Params:
+        cve_id = factory.LazyAttribute(lambda f: "CVE-2024-0001")
         ext_id = factory.LazyAttribute(
-            lambda f: "CVE-2023-0001" if f.source == Snippet.Source.NVD else "GHSA-0001"
-        )
-        cve_id = factory.LazyAttribute(
-            lambda f: "CVE-2023-0001"
-            if f.source == Snippet.Source.NVD
-            else "CVE-2024-0001"
+            lambda f: f.cve_id if f.source == Snippet.Source.NVD else "GHSA-0001"
         )
         url = factory.LazyAttribute(
-            lambda f: "https://nvd.nist.gov/vuln/detail/CVE-2023-0001"
+            lambda f: f"https://nvd.nist.gov/vuln/detail/{f.ext_id}"
             if f.source == Snippet.Source.NVD
-            else "https://osv.dev/vulnerability/GHSA-0001"
+            else f"https://osv.dev/vulnerability/{f.ext_id}"
         )
 
     source = factory.Faker(
         "random_element", elements=[Snippet.Source.NVD, Snippet.Source.OSV]
     )
-    external_id = factory.LazyAttribute(lambda f: f.ext_id)
+
+    external_id = factory.LazyAttribute(
+        lambda f: f.ext_id
+        if f.source == Snippet.Source.NVD or f.cve_id is None
+        else f"{f.ext_id}/{f.cve_id}"
+    )
 
     @factory.lazy_attribute
     def content(self):
+        # contains all NVD fields and only the currently used OSV fields
         data = {
             "cve_id": self.cve_id,
             "cvss_scores": [
                 {
                     "score": 8.1,
-                    "issuer": FlawCVSS.CVSSIssuer.NIST,
+                    "issuer": FlawCVSS.CVSSIssuer.REDHAT,
```

Comment:
nitpick: That sounds a bit weird that the source is NVD and the CVSS is issued by Red Hat.

Reply:
Also discussed in person.